src/screens/AddSiteFormScreen.js

```diff
@@ -335,86 +370,140 @@ class AddSiteFormScreen extends Component {
+                    <Text style={labelStyle}>{accessibility}</Text>
                     <Picker
+                        style={[formInputStyle, pickerStyle]}
                         selectedValue={accessibilityOption}
                         onValueChange={this.onUpdateAccessibilityOption}
                     >
                         {this.accessibilityOptions()}
                     </Picker>
 
-                    <FormLabel>{price}</FormLabel>
+                    <Text style={labelStyle}>{price}</Text>
                     <Picker
+                        style={[formInputStyle, pickerStyle]}
                         selectedValue={priceOption}
                         onValueChange={this.onUpdatePriceOption}
                     >
                         {this.priceOptions()}
                     </Picker>
 
-                    <FormLabel>{facilities}</FormLabel>
-                    <View>
+                    <Text style={labelStyle}>{facilities}</Text>
+                    <View style={formInputStyle}>
                         {this.renderCheckboxes(facilities_options)}
                     </View>
 
-                    <FormLabel>{features}</FormLabel>
-                    <View>
+                    <Text style={labelStyle}>{features}</Text>
+                    <View style={formInputStyle}>
                         {this.renderCheckboxes(features_options)}
                     </View>
 
+                    <Text style={labelStyle}>{alternate_sites} {optional}</Text>
+                    <Input
+                        containerStyle={formInputStyle}
+                        placeholder={alternate_sites_placeholder}
+                        value={siteAlternateSitesText}
+                        onChangeText={this.onUpdateAlternateSitesText}
+                        containerStyle={largeTextInput}
+                        blurOnSubmit={true}
+                        multiline={true}
+                        autoGrow={true}
+                        editable={true}
+                    />
+
+                    <Text style={labelStyle}>{cell_service} {optional}</Text>
+                    <View style={cellServiceContainerStyle}>
+                        <Picker
+                            style={[formInputStyle, cellProviderPickerStyle]}
+                            selectedValue={cellProviderOption}
+                            onValueChange={this.onUpdateCellProviderOption}
+                        >
+                            {this.cellProviderOptions()}
+                        </Picker>
+
+                        <Picker
+                            style={[formInputStyle, cellStrengthPickerStyle]}
+                            selectedValue={cellStrengthOption}
+                            onValueChange={this.onUpdateCellStrengthOption}
+                        >
+                            {this.cellStrengthOptions()}
+                        </Picker>
+                    </View>
```

Comment:
I put our optional fields at the bottom of the form. We'll probably want these hidden by an accordion by default so the user can expand them only if they actually want to dive into optional fields.